LayoutTests/imported/w3c/web-platform-tests/css/css-lists/parsing/counter-set-valid.html

```diff
@@ -15,9 +15,6 @@
 test_valid_value('counter-set', 'chapter', 'chapter 0');
 test_valid_value('counter-set', 'section -1');
 test_valid_value('counter-set', 'first -1 second third 99', 'first -1 second 0 third 99');
-test_valid_value('counter-set', 'section calc(1)');
```

Comment:
@weinig - These were not synced to upstream? I will undo them for time being since I have to add platform specific expectations any way on another push.